src/blob/index/bptree/node.rs

```diff
@@ -53,10 +55,10 @@ impl Node {
         Err(l as usize)
     }
 
-    pub(super) fn key_offset_serialized(buf: &[u8], key: &[u8]) -> Result<u64> {
+    pub(super) fn key_offset_serialized<K: Key>(buf: &[u8], key: &K) -> Result<u64> {
```

Comment:
Please, take a look at `key_offset` method. It uses `binary_search_by` on keys, so it needs to be updated, but additionally it is marked with `#[allow(dead_code)]` so the method should be inspected. There are two ways of resolution: update method or remove it.

Reply:
`key_offset` method is used only in tests, but still I've updated it with simple conversions